### PR TITLE
Removes TGS2 remnants

### PR DESCRIPTION
**Pull request by Cyberboss**
With both servers running I think it's time.

TGS2 will still work, but it loses support for in game PR details and classic IRC stuff

### CODE REVIEW

**Comment by kevinz000** [comment]
//kevinz000 if you touch this I will hunt you down

File: code/__HELPERS/unsorted.dm [hunk 1]
```diff
@@ -1345,63 +1345,6 @@ GLOBAL_DATUM_INIT(dview_mob, /mob/dview, new)
 				return FALSE
 	return TRUE
 
-//WHATEVER YOU USE THIS FOR MUST BE SANITIZED TO SHIT, IT USES SHELL
-//It also sleeps
-
-//Set this to TRUE before calling
-//This prevents RCEs from badmins
-//kevinz000 if you touch this I will hunt you down
```
Comment (kevinz000) [inline code/__HELPERS/unsorted.dm]: :joy: :joy_cat: :joy: :joy_cat: :joy: :joy_cat: :joy: :joy_cat: :joy: :joy_cat: :joy: :joy_cat: :joy: :joy_cat: :joy: :joy_cat: :joy: :joy_cat: :joy: :joy_cat: :joy: :joy_cat: :joy: :joy_cat: :joy: :joy_cat: :joy: :joy_cat: :joy: :joy_cat: :joy: :joy_cat: :joy: :joy_cat: :joy: :joy_cat: :joy: :joy_cat: :joy: :joy_cat: :joy: :joy_cat: :joy: :joy_cat: :joy: :joy_cat: :joy: :joy_cat: :joy: :joy_cat: :joy: :joy_cat: :joy: :joy_cat: :joy: :joy_cat: :joy: :joy_cat: :joy: :joy_cat: :joy: :joy_cat: :joy: :joy_cat: :joy: :joy_cat: :joy: :joy_cat: :joy: :joy_cat: :joy: :joy_cat:

**Comment by Shadowlight213** [comment]
Rude

**Comment by Shadowlight213** [comment]
Well. I WAS going to rewrite my bot to use the tgs3 interface. However, cyberboss made it use a separate comms key, which regenerates every service restart.

**Comment by optimumtact** [comment]
Why is this tagged do not merge?

**Comment by Cyberboss** [comment]
Pending 3.1 release of TGS3

**Comment by Cyberboss** [comment]
Might as well leave it for now

**Comment by MrStonedOne** [comment]
you might want to keep the irc bot, its seperate from the tgns interface, doesn't require the tgns2 interface, and it existed long before the tgns interface.

**Pull request closed without merging**